src/gui/static/src/app/components/pages/wallets/wallet-detail/wallet-detail.component.ts

```diff
@@ -54,6 +55,11 @@ export class WalletDetailComponent {
     setTimeout(function () {
       address.copying = false;
     }, 500);
+  }
 
+  showQrCode(address: string) {
+    const config = new MatDialogConfig();
+    config.data = { address };
```

Comment:
Don't you have to add a key for address? Does this compile or work?

Reply:
I send it there as a string, not an object
https://github.com/skycoin/skycoin/pull/1185/files/10b612704520814a8212effa9cd6abc83f332d02#diff-16f295f5e79f9cbf7bb4baf688e19cdcR5

Reply:
Why not remove brackets?

Reply:
QRCode component expects an object (it might receive more parameters in the future)